### PR TITLE
[web-animations-1][editorial] Reformat

### PR DESCRIPTION
**Pull request by fantasai**
* Apply semantic line breaks throughout https://rhodesmill.org/brandon/2012/one-sentence-per-line/
* Fix indentation
* Fix/update markup
* Fix punctuation (commas, etc.)

* Live: https://fantasai.inkedblade.net/style/specs/web-animations-1/Overview
* Diff: https://services.w3.org/htmldiff?doc1=&doc2=https%3A%2F%2Ffantasai.inkedblade.net%2Fstyle%2Fspecs%2Fweb-animations-1%2FOverview

### CODE REVIEW

**Comment by birtles** [comment]
Looks amazing! Thank you so much! I looked over all the changes and they all look great.

I tried to build it locally, however, and got a few errors:

```
/home/birtles/.local/lib/python3.10/site-packages/html5lib/_ihatexml.py:266: DataLossWarning: Coercing non-XML name: dfnlt="setkeyframes(keyframes)"
  warnings.warn("Coercing non-XML name: %s" % name, DataLossWarning)
WARNING: The following <var>s were only used once in the document:
  'timeline', in algorithm 'animation time to origin-relative time'
If these are not typos, please add an ignore='' attribute to the <var>.
LINK ERROR: No 'dfn' refs found for 'active time space'.
[=active time space=]
LINE ~3075: No 'dfn' refs found for 'document time space'.
[=document time space=]
LINE ~5184: No 'dfn' refs found for 'iteration index'.
[=iteration index=]
```

Any chance you can look into these? For example, I think these edits assumed that "active time space" was a definition when it was simply a quasi-reference to an earlier paragraph.

**Comment by fantasai** [comment]
@birtles Heh, I'd left those errors as a reminder to fix up the cross-linking of those terms at some point. :) But yeah, I can make it build cleanly. Done.

File: web-animations-1/Overview.bs [hunk 1]
```diff
@@ -233,6255 +234,7011 @@ spec:infra; type:dfn; for:list; text:extend
...
+  the offset is greater than or equal to
+  the offset of the previous [=keyframe=] in the list
+  with a [=keyframe offset=] that is not null, if any.
+
+  The behavior when [=keyframes=] overlap
+  or have unsupported values is defined in <a
+  href="#the-effect-value-of-a-keyframe-animation-effect"
+  section></a>.
+
+  Each keyframe also has a [=timin
```
Comment (birtles) [inline web-animations-1/Overview.bs:5787]:
```suggestion
  : <dfn lt="setKeyframes(keyframes)">void setKeyframes(object? keyframes)</dfn>
```

**Review by birtles — APPROVED** [review]
Looks good with the one suggestion applied.

Although it looks like GitHub might be having trouble showing the diff of the suggestion correctly since it's such a big diff.

For posterity the change is:

```diff
-  : <dfnlt="setKeyframes(keyframes)">void setKeyframes(object? keyframes)</dfn>
+  : <dfn lt="setKeyframes(keyframes)">void setKeyframes(object? keyframes)</dfn>
```

**Comment by fantasai** [comment]
@birtles Wow I'm impressed you caught that :D Fixed.

**Pull request merged**